### DISCUSSION TITLE
Enforce naming convention in the modules

### DISCUSSION THREAD

**Discussion by marianoc84** [terraform-linters/tflint#2083] [Q&A] [answered]
I have this file tree: 

```
.
├── locals.tf
├── main.tf
├── modules
│   └── vpc
│       ├── locals.tf
│       ├── main.tf
│       └── variables.tf
├── outputs.tf
├── providers.tf
├── .tflint.hcl
└── variables.tf
```

I have written a _.tflint.hcl_ as the following:
```
plugin "terraform" {
    enabled = true
    preset  = "all"
}

plugin "aws" {
    enabled = true
    version = "0.32.0"
    source  = "github.com/terraform-linters/tflint-ruleset-aws"
}

rule "terraform_naming_convention" {
    enabled = true
}
```
and this is in the root folder of my terraform. If I run `tflint --recursive` and in _modules/vpc/main.tf_ there's a wrong naming convention for a resource (e.g. `myNewResource`) nothing happen; but, if I copy the same _.tflint.hcl_ file in the _vpc_ directory the notice is shown correctly.

How can I enforce the naming convention in the module avoiding polutting the code with tons  of _.tflint.hcl_ files?

**Comment by bendrucker** [comment, marked as answer]
Use an absolute path if you don't want TFLint to try to load configuration relative to each directory:

https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/config.md#configuring-tflint

By default, `.tflint.hcl` is loaded relative to the module under test. But if you pass an absolute path to the file at your repo root, it'll always load the same config.
